Loop.xcodeproj/xcshareddata/xcschemes/Complication - WatchApp.xcscheme

```diff
@@ -88,18 +88,16 @@
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES"
       launchAutomaticallySubstyle = "32">
-      <RemoteRunnable
-         runnableDebuggingMode = "2"
-         BundleIdentifier = "com.apple.Carousel"
-         RemotePath = "/$(MAIN_APP_DISPLAY_NAME)">
+      <BuildableProductRunnable
```

Comment:
These `.xcscheme` files constantly are dirtied by doing a build locally.  Maybe because I'm running a newer version of Xcode? (12.5)  I'm hoping by checking these in this will stop it.